readme.md

```diff
@@ -59,30 +54,28 @@ Getting the code (optionally, you can create a workspace):
     export PATH=$PATH:$GOPATH/bin
     go get -t github.com/zalando/skipper
 
-Build:
+#### Building
+[needs more context copy]
```

Comment:
"Building skipper happens using the standard go tooling. To build the shipped skipper executable without custom filters or predicates, one simple calls `go install` on the `cmd/skipper` package: